schema/post_mutations.graphql

```diff
@@ -0,0 +1,5 @@
+extend type Mutation {
+    createPost(input: PostCreateInput!): Post!
+    updatePost(input: PostUpdateInput): Post!
```

Comment:
Why is the PostUpdateInput not compulsory?

Reply:
Just like in `User` mutation, if there is null input we can simply return existing entity.
so I followed the same in `Post` and `Author`

Reply:
If we are doing an update mutation I think it doesn't make sense to not send any input. How will the update resolver know which post to update?

Reply:
On null update/delete Input, we simply return the entity unaltered

Will that be good?

Reply:
But how will you identify which entity to return since you are not sending any ID.

Reply:
Yeah, make sense, we would need the input data, I forgot the fact that in User's case the ID was stored in the `Request Context` where as here in Post and Author we are not going to store that in request context [For now atleast]


I will add the compulsion